apps/docs/libraries/ts/sdk/ratelimits/overrides/list-overrides.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: "List Overrides"
+description: "Lists all overrides"
+---
+
+## Request
+
+<Warning>
+Either `namespaceId` or `namespaceName` is required. Not both.
+</Warning>
+
+<ParamField body="namespaceId" type="string">
+The id of the namespace. Either namespaceId or namespaceName must be provided
+</ParamField>
+
+<ParamField body="namespaceName" type="string">
+Namespaces group different limits together for better analytics. You might have a namespace for your public API and one for internal tRPC routes. Wildcards can also be used, more info can be found at https://www.unkey.com/docs/ratelimiting/overrides#wildcard-rules
+</ParamField>
+
+## Response
+
+<ResponseField name="result">
+<Expandable title="properties" defaultOpen>
+
+<ResponseField name="overrides" type="Array" required>
+
+ <Expandable>
+  <ParamField body="id" type="string" required>
+Identifier of the override requested
+</ParamField>
+
+<ParamField body="identifier" type="string" required>
+Identifier of your user, this can be their userId, an email, an ip or anything else. Wildcards ( * ) can be used to match multiple identifiers, More info can be found at https://www.unkey.com/docs/ratelimiting/overrides#wildcard-rules
+</ParamField>
+
+<ParamField body="limit" type="number" required>
+How many requests may pass in a given window.
+</ParamField>
+
+<ParamField body="duration" type="number" required>
+The window duration in milliseconds.
+</ParamField>
+
+<ParamField body="async" type="boolean">
+Async will return a response immediately, lowering latency at the cost of accuracy.
+</ParamField>
+
+ </Expandable>
+</ResponseField>
+
+<ResponseField name="total" type="number" required>
+The total number of overrides
+</ResponseField>
+<ResponseField name="cursor" type="string">
+The cursor to use for pagination
+</ResponseField>
+ </Expandable>
+
+</ResponseField>
+
+
+<RequestExample>
+  
+```ts
+const override = await unkey.ratelimit.listOverrides({
```

Comment:
plural please

Reply:
I don't understand this tbh, I just marked this and even gave you a committable example
![CleanShot 2024-12-12 at 19 42 26@2x](https://github.com/user-attachments/assets/533cc7b3-97a3-4b73-b766-16e8f9e361f7)